.golangci.yaml

```diff
@@ -12,7 +12,6 @@ linters:
   enable:
     - asciicheck
     - bodyclose
-    - depguard
```

Comment:
This has been moved to disabled until we make a config for allowed/denied dependencies - it was previously not working and only surfaces an error in the latest release